webrtc.html

```diff
@@ -1808,14 +1808,50 @@
                     <code>true</code>, abort these steps.</p>
                   </li>
                   <li>
-                    <p>Destroy <var>connection</var>'s <a>ICE Agent</a>,
-                    abruptly ending any active ICE processing and any active
-                    streaming, and releasing any relevant resources (e.g. TURN
-                    permissions).</p>
+                    <p>
+                      Destroy <var>connection</var>'s <a>ICE Agent</a>,
+                      abruptly ending any active ICE processing and any active
+                      streaming, and releasing any relevant resources (e.g. TURN
+                      permissions).
+                    </p>
                   </li>
-                  <li>All <code><a>RTCRtpSender</a></code>s in
-                  <var>connection</var>'s <a>set of senders</a> are now
-                  considered <a>stopped</a>.
+                  <li>
+                    <p>
+                      For all <code><a>RTCRtpSender</a></code>s in
+                      <var>connection</var>'s <a>set of senders</a>,
+                      set <code>RTCRtpSender.transport.state</code> and
```

Comment:
Instead of the "static looking" style ```RTCRtpSender.transport.state```, that we have removed elsewhere, we could write something like:
For every ```RTCRtpSender```, _sender_, in _connection_'s set of senders, set _sender.transport.state_...